### ISSUE TITLE
Flex flavors for B3/C3/R3 compute ranges

### ISSUE THREAD

**Issue by JacquesMrz** [ovh/public-cloud-roadmap#545]
### Summary

For classic flavors (models), only resize to a higher flavors (in terms of disk space) is possible.
_-Flex_ flavors allow resizing to higher or lower models with a single disk size.

### Intended Outcome

_-Flex_ flavors available for B3 / C3 / R3 ranges.
_-Flex_ flavors prices are the same as for the equivalent classic flavors.

### How will it work

[ Resize an instance via the OVHcloud Control Panel](https://help.ovhcloud.com/csm/en-public-cloud-compute-control-panel-resize-instance?id=kb_article_view&sysparm_article=KB0051118)
[ Resize an instance via Horizon](https://help.ovhcloud.com/csm/en-public-cloud-compute-horizon-resize-instance?id=kb_article_view&sysparm_article=KB0051133)

**Comment by hbrombeer** [comment]
Is there a plan for higher cpu clockspeed instances?

**Comment by mhurtrel** [comment]
Hi @hbrombeer There is no plan for higher frequency cores in the short term. But my colleagues @JacquesMrz  and @yomovh insist that though they show lower clock speed "on the paper" these latest generation offer better single core performance than, for example, our legacy C2 instances. Have you tested them ? If so and if the single core performance still is not enough, we would be interested to get full details on your use case and a type of CPU that offer the performance you expect.

**Comment by hbrombeer** [comment]
No i have not tested them yet, we want to use the ovh cloud plattform for gameserver deployments so every hz counts 😂 I will test it in the coming weeks and then i will come back to you.

**Comment by yomovh** [comment]
@hbrombeer the geekbench results  mentionned by @mhurtrel:
- [C2-7](https://browser.geekbench.com/v6/cpu/6767311) 
- [C3-8](https://browser.geekbench.com/v6/cpu/6767297) 

TL;DR even if the C2 is 3.1GHz and the C3 is 2.4GHz, the C3 single core score is way higher than the C2 (1412 vs 990)
